src/docfx/lib/github/UserProfileCache.cs

```diff
@@ -70,6 +86,51 @@ public static UserProfileCache Create(string cachePath, GitHubAccessor github)
             }
         }
 
+        private async Task<List<Error>> UpdateCacheByCommit(GitCommit commit, Repository repo)
+        {
+            var errors = new List<Error>();
+            var author = commit.AuthorEmail;
+            if (string.IsNullOrEmpty(author) || GetByUserEmail(author) != null)
+                return new List<Error>();
+
+            var (getNameErrors, authorName) = await _github.GetNameByCommit(repo.Owner, repo.Name, commit.Sha);
+            errors.AddRange(getNameErrors);
```

Comment:
>errors [](start = 12, length = 6)

this is a pre-populate process, I feel like we don' need to log errors here.

Reply:
I want users to know the result is partial due to some GitHub issue like rate limit exceeded.

---
In reply to: [211854678](https://github.com/dotnet/docfx/pull/3176#discussion_r211854678) [](ancestors = 211854678)